.circleci/config.yml

```diff
@@ -83,6 +89,7 @@ jobs:
       - install_graph_cli
       - build_sub_graph
       - deploy_master
+      - deploy_gke_master
```

Comment:
ya, this was only changes into master branch but didn't committed back to develop/release that's why this changes.